Source/ApiTemplate/Source/ApiTemplate/Commands/GetCarPageCommand.cs

```diff
@@ -72,41 +72,41 @@
                     HasNextPage = hasNextPage,
                     HasPreviousPage = hasPreviousPage,
                     NextPageUrl = hasNextPage ? new Uri(this.linkGenerator.GetUriByRouteValues(
-                        this.httpContextAccessor.HttpContext,
+                        this.httpContextAccessor.HttpContext!,
```

Comment:
Perhaps it might be a good idea to move this into a local variable?

var context = this.httpContextAccessor.HttpContext!;